src/cljs/ataru/virkailija/application/application_subs.cljs

```diff
@@ -885,6 +885,15 @@
   (fn [db _]
     (get-in db [:editor :user-info :superuser?])))
 
+(re-frame/reg-sub
```

Comment:
Mietin et näitä authorisointiin liittyviä subeja voisi siirtää omaan filuunsa, esim application_authorization_subs

Reply:
siirretty!